src/SFA.DAS.EmployerIncentives.Api/Controllers/AccountController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.EmployerIncentives.Api.Extensions;
+using SFA.DAS.EmployerIncentives.Interfaces;
+using SFA.DAS.EmployerIncentives.Models.PassThrough;
+
+namespace SFA.DAS.EmployerIncentives.Api.Controllers
+{
+    [ApiController]
+    public class AccountController : ControllerBase
+    {
+        private readonly IPassThroughApiClient _client;
+
+        public AccountController(IPassThroughApiClient client)
+        {
+            _client = client;
+        }
+
+        [HttpGet]
+        [Route("/accounts/{accountId}/legalentities")]
+        public async Task<IActionResult> GetLegalEntities(long accountId)
+        {
+            var innerApiResponse = await _client.Get($"/accounts/{accountId}/legalentities");
```

Comment:
Is the purpose of CreateObjectResult to just set the header?  If you use HttpCompletionOption.ResponseHeadersRead as an option then the headers can be read without loading the body.  Then the body should be able to be streamed to the client without loading in memory.